### ISSUE TITLE
cluster routes with websockets?

### ISSUE THREAD

**Issue by matti** [nats-io/nats-server#3509]
Can I configure cluster to use websockets - I'd like my nodes to connect over ws instead of tcp.

Tried
```
cluster {
  routes: [
    ws://server:8080
  ]
```

And I get 
```
 Router parser ERROR, state=18, i=1: proto='"TTP/1.1 400 Bad Request\r\nContent"...'
```

**Comment by ripienaar** [comment]
Websockets only for leafnodes and clients at the moment.

**Comment by matti** [comment]
Implemented a workaround with https://github.com/jpillora/chiselOn 6. Jan 2023, at 18.49, Derek Collison ***@***.***> wrote:﻿
Closed #3509 as completed.
